frontend/src/pages/Plants/index.tsx

```diff
@@ -18,6 +20,9 @@ const PlantsPage = () => {
 		const fetchData = async () => {
 			if (householdId) {
 				const plants = await getPlants(householdId);
+				await getHousehold(householdId).then((name) => {
```

Comment:
For the household name, you dont need to fetch. Just get from the households context. Filter by id and get the name.